### ISSUE TITLE
Generator not working, missing bootstrap-affix.js

### ISSUE THREAD

**Issue by flexd** [romainberger/yeoman-flask#8]
Hey. Just installed Yeoman on a new machine with 'npm install -g yo'  as well as 'npm install -g generator-flask'

However running this I get the following error:

``` python

kristoffer at beastmode in ~/code/cognitive.io
$ yo flask
[?] Name of the app: app
[?] Would you like to include Twitter Bootstrap for Compass instead of CSS? No
[?] Would you like to include the Twitter Bootstrap JS plugins? Yes
[?] Do you want to use Frozen-Flask to build a static version of the app? No

fs.js:427
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory '/home/kristoffer/.cache/yeoman/twitter/bootstrap/v2.1.0/js/bootstrap-affix.js'
    at Object.fs.openSync (fs.js:427:18)
    at Object.fs.readFileSync (fs.js:284:15)
    at Generator.copy (/home/kristoffer/.nvm/v0.10.15/lib/node_modules/generator-flask/node_modules/yeoman-generator/lib/actions/actions.js:96:13)
    at Object.copy (/home/kristoffer/.nvm/v0.10.15/lib/node_modules/generator-flask/node_modules/yeoman-generator/lib/actions/actions.js:369:12)
    at /home/kristoffer/.nvm/v0.10.15/lib/node_modules/generator-flask/app/index.js:77:16
    at Array.forEach (native)
    at /home/kristoffer/.nvm/v0.10.15/lib/node_modules/generator-flask/app/index.js:75:8
    at done (/home/kristoffer/.nvm/v0.10.15/lib/node_modules/generator-flask/node_modules/yeoman-generator/lib/actions/actions.js:391:5)
    at /home/kristoffer/.nvm/v0.10.15/lib/node_modules/generator-flask/node_modules/yeoman-generator/lib/actions/actions.js:340:16
    at Object.oncomplete (fs.js:107:15)
```

**Comment by romainberger** [comment]
Crap. Forgot to get back to your issue, sorry. I'll take a look asap.
Seems to come from Yeoman itself as it's an error coming from the cache. Not sure tho. Are you running the last version of Yeoman?

**Comment by flexd** [comment]
Yeah I just installed yeoman with the command specified above not 5 minutes before.

**Comment by suenot** [comment]
I have the same error. yeoman 1.0.4, node 0.10.6, npm 1.2.18, ubuntu 12.04.

**Comment by jamiis** [comment]
ditto. want to track this.

**Comment by gianpaj** [comment]
same here

```
$ yo --version
1.0.4
$ node --version
v0.10.12
$ npm --version
1.2.32
$ npm list generator-flask
npm WARN package.json backbone@0.9.2 No repository field.
npm WARN package.json ender-js@0.4.3-dev No repository field.
npm WARN package.json assert-plus@0.1.2 No repository field.
npm WARN package.json ctype@0.5.2 No repository field.
npm WARN package.json cheerio-select@0.0.3 No repository field.
/Users/GianPaJ/tmp
└── generator-flask@1.2.1

OS X 10.8.4

$ yo flask
[?] Name of the app: flasknameapp
[?] Would you like to include Twitter Bootstrap for Compass instead of CSS? Yes
[?] Would you like to include the Twitter Bootstrap JS plugins? Yes
[?] Do you want to use Frozen-Flask to build a static version of the app? No

     info ... Fetching http://github.com/twitter/bootstrap/archive/v2.1.0.tar.gz ...
     info This might take a few moments
.
✔ Done in 0.803s.
✔ Done in /Users/gianpaj/.cache/yeoman/twitter/bootstrap/v2.1.0


fs.js:427
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory '/Users/gianpaj/.cache/yeoman/twitter/bootstrap/v2.1.0/js/bootstrap-affix.js'
    at Object.fs.openSync (fs.js:427:18)
    at Object.fs.readFileSync (fs.js:284:15)
    at Generator.copy (/Users/GianPaJ/tmp/node_modules/generator-flask/node_modules/yeoman-generator/lib/actions/actions.js:102:13)
    at Object.copy (/Users/GianPaJ/tmp/node_modules/generator-flask/node_modules/yeoman-generator/lib/actions/actions.js:398:12)
    at /Users/GianPaJ/tmp/node_modules/generator-flask/app/index.js:77:16
    at Array.forEach (native)
    at /Users/GianPaJ/tmp/node_modules/generator-flask/app/index.js:75:8
    at done (/Users/GianPaJ/tmp/node_modules/generator-flask/node_modules/yeoman-generator/lib/actions/actions.js:420:5)
    at Extract.<anonymous> (/Users/GianPaJ/tmp/node_modules/generator-flask/node_modules/yeoman-generator/lib/actions/fetch.js:80:7)
    at Extract.EventEmitter.emit (events.js:117:20)
```

**Comment by romainberger** [comment]
Thanks for the logs. 
Makes me realize it's using bootstrap 2.1.0, I should update to the latest version at the same time.
Will look into this asap

**Comment by romainberger** [comment]
Fixed with #10 thanks to @halkeye.
